framework/quota/src/main/java/org/apache/cloudstack/quota/QuotaManagerImpl.java

```diff
@@ -340,7 +340,7 @@ public QuotaUsageVO updateQuotaDiskUsage(UsageVO usageRecord, final BigDecimal a
             BigDecimal quotaUsgage;
             BigDecimal onehourcostpergb;
             BigDecimal noofgbinuse;
-            onehourcostpergb = tariff.getCurrencyValue().multiply(aggregationRatio);
+            onehourcostpergb = tariff.getCurrencyValue().divide(s_hoursInMonth, 8, RoundingMode.HALF_DOWN);
```

Comment:
I think you need to remove the parameter aggregationRotio` on this change as it is no longer used is it?